apps/dialtone-documentation/docs/components/empty-states.md

```diff
@@ -0,0 +1,378 @@
+---
+title: Empty state
+description: When there's no data to display in a list, table, or other information container, we use an ‘Empty State’ component to guide users, clarifying the expected information and suggesting ways to populate it.
+status: ready
+thumb: true
+image: assets/images/components/empty-state.png
+figma: planned
+storybook: https://dialtone.dialpad.com/vue/?path=/story/components-card--default
+---
+
+<code-well-header>
+  <dt-empty-state
+    illustration-name="mind"
+    header-text="Nothing to see here"
+    body-text="Looks like there's no data to display here."
+  >
+    <template #body>
+      <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+        <button class="d-btn">Click me</button>
+        <button class="d-btn d-btn--primary">Click me</button>
+      </div>
+    </template>
+  </dt-empty-state>
+</code-well-header>
+
+## Morphology & Anatomy
+
+Image with bullets
+<code-well-header>
+<img class="d-w90p" alt="empty state bullets" src="/assets/images/components/empty-state01.png">
+</code-well-header>
+
+1. Illustrations*
+2. Title
+3. Description*
+4. Actions
+
+- Primary (this primary action, can be a primary button or a muted one)
+- Secondary*
```

Comment:
To nest an unordered list under an ordered list I think you have to put 3 spaces of indentation before it (super weird markdown thing). Right now it looks kinda off.

![Screenshot 2024-05-28 at 12 12 57 PM](https://github.com/dialpad/dialtone/assets/64808812/6e338c90-eefe-4c13-9c0c-96bf8b69422c)

Reply:
Updated

<img width="540" alt="image" src="https://github.com/dialpad/dialtone/assets/89984179/b495e9e7-6c0e-4856-99f0-7ae1c413f5a1">